corehq/apps/couch_sql_migration/management/commands/migrate_domain_from_couch_to_sql.py

```diff
@@ -172,7 +174,30 @@ def handle(self, domain, **options):
                 toggles.DATA_MIGRATION.enable(domain)  # Prevent any more changes on domain
                 set_couch_sql_migration_not_started(domain)
                 set_couch_sql_migration_not_started(dst_domain)
-                # TODO: Reindex forms & cases
+                for case in self._iter_migrated_cases(domain, dst_domain):
+                    publish_case_saved(case, send_post_save_signal=True)
+
+    def _iter_migrated_cases(self, src_domain, dst_domain):
+        db = get_diff_db(src_domain)
+        ZERO = Counts(0, 0)
+        if db.has_doc_counts():
+            doc_counts = db.get_doc_counts()
+        else:
+            doc_counts = None
+        for doc_type in CASE_DOC_TYPES:
+            if doc_counts is not None:
+                counts = doc_counts.get(doc_type, ZERO)
+                case_ids = counts
```

Comment:
was this assignment intentional? seems like `chunked(case_ids, 500)` will not return case ids with that value

Reply:
Nope! That is 100% a bug. (Also, this should not be a method.)